suite/tests/client-interface/memory_dump_test.templatex

```diff
@@ -14,13 +14,13 @@ ELF Header:
   Type:                              CORE \(Core file\)
   Machine:                           .*
   Version:                           0x1
-  Entry point address:               0x0
+  Entry point address:               0x[0-9a-f]+
   Start of program headers:          64 \(bytes into file\)
   Start of section headers:          [0-9]+ \(bytes into file\)
   Flags:                             0x0
   Size of this header:               64 \(bytes\)
   Size of program headers:           56 \(bytes\)
-  Number of program headers:         1
+  Number of program headers:         [0-9]+
```

Comment:
Please include a sample output of `readelf -l` on a coredump file in the PR description to help show this was further tested and the result looks reasonable (at some point in the future we'll have a test that launches the file I assume).

Reply:
I've added the output of readelf -l to the PR description.

The current memory dump file is using the core file format to store register values. In order to launch the file, we need a converter to convert the memory dump file to either an executable or shared object type.